### PR TITLE
Camera forward example by Cat-Robot

### PR DESCRIPTION
**Pull request by Midhil457**
Example by @Cat-Robot
PR for #16 

The example has been reviewed and the needed changes have been made

This example shows how to make the camera lean/face towards the direction of the player, making the path more visible

### CODE REVIEW

File: examples/forward-camera/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+A platformer camera that leans towards the player's direction. Making the obstacles more visible
```
Comment (4ian) [inline examples/forward-camera/README.md:1]: "A platformer camera that leans toward the direction the player is looking at. Useful to give more visibility to the player when running through a level."

(careful about your original description, it does not include a period at the end, and you can't start a sentence with "Making the [...]")

**Comment by 4ian** [comment]
Thanks for taking the time to review this! :) Added one comment and then good to go :)

**Comment by Midhil457** [comment]
My bad, fixed it.

Thanks

**Comment by 4ian** [comment]
Thank you!!

**Pull request merged**